bindings/wasm/examples/src/1_advanced/6_domain_linkage.ts

```diff
@@ -0,0 +1,176 @@
+    console.log("Configuration Resource:", JSON.stringify(configurationResource.toJSON(), null, 2));
+
+    // Now the DID Document links to the Domains through the service, and the Foo domain links to the DID
+    // through the DID Configuration resource. A bidirectional linkage is established.
+    // Note however that bidirectionality is not a hard requirement. It is valid to have a Domain Linkage
+    // credential point to a DID, without the DID having a service that points back.
+
+    // =====================================================
+    // Verification can start from two different places.
+    // The first case answers the question "What DID is this domain linked to?"
+    // while the second answers "What domain is this DID linked to?".
+    // ====================================================
+
+    // =====================================================
+    // → Case 1: starting from domain
+    // =====================================================
+
+    // Fetch the DID Configuration resource (For example using the Fetch API).
+    // Note that redirection must be disabled when fetching.
+    const _configurationUrl = `${domainFoo}/.well-known/did-configuration.json")`;
+
+    // But since the DID Configuration
+    // resource isn't available online in this example, we will simply use the JSON.
+    let fetchedConfigurationResource = DomainLinkageConfiguration.fromJSON(configurationResource);
+
+    // Retrieve the issuers of the Domain Linkage Credentials which correspond to the possibly linked DIDs.
+    let issuers: Array<string> = fetchedConfigurationResource.issuers();
+    const issuerDocument: IotaDocument = await didClient.resolveDid(IotaDID.parse(issuers[0]));
+
+    // Validate the linkage between the Domain Linkage Credential in the configuration and the provided issuer DID.
+    // Validation is succeeds when no error is thrown.
+    DomainLinkageValidator.validateLinkage({
+        domain: domainFoo,
+        configuration: fetchedConfigurationResource,
+        issuer: issuerDocument,
+        validationOptions: CredentialValidationOptions.default(),
+    });
+
+    // =====================================================
+    // → Case 2: starting from a DID
+    // =====================================================
+
+    const didDocument: IotaDocument = await didClient.resolveDid(did);
+
+    // Get the Linked Domain Services from the DID Document.
+    let linkedDomainServices: LinkedDomainService[] = didDocument
+        .service()
+        .filter(service => LinkedDomainService.isValid(service))
+        .map(service => LinkedDomainService.fromService(service));
+
+    // Get the domains included in the Linked Domain Service.
+    let domains: string[] = linkedDomainServices[0].domains();
+
+    // Fetch the DID Configuration resource (For example using the Fetch API).
+    // Note that redirection must be disabled when fetching.
```

Comment:
Should mention why and that it is an implementers decision.

Reply:
The main issue here was that Fetch API has different implementations for Node.js and the browser. But maybe we can still add this feature in the future. I might create an issue for that.

Reply:
We should explain why it should be disabled.

Reply:
I understand you now, It's in the specs, I'll add that.